driver/csiplugin/controllerserver.go

```diff
@@ -635,6 +635,28 @@ func (cs *ScaleControllerServer) CreateVolume(ctx context.Context, req *csi.Crea
 
 	glog.Infof("volume:[%v] -  spectrum scale volume create params : %v\n", scaleVol.VolName, scaleVol)
 
+	if scaleVol.IsFilesetBased && scaleVol.Compression != "" {
```

Comment:
Also since we need specific releases for partition we will need to add version check

Reply:
Compression we don't need a check since it does not make use of the new partition features. We only need it for tiering, but I will definitely add the check on fs version for that.